Android.bp

```diff
@@ -0,0 +1,67 @@
+cc_library_headers {
```

Comment:
Do not introduce third-party compilation scripts. The correct way should be to export the SDK from Nuttx and compile libc from the out of tree.

Reply:
> Do not introduce third-party compilation scripts. The correct way should be to export the SDK from Nuttx and compile libc from the out of tree.

I can choose not to submit this Android.bp, but I feel this is an opportunity to integrate NuttX into the AOSP build process. If we were to use the SDK approach, it would involve issues with compilation parameters and toolchains, so I directly used the source code for compilation.

Reply:
This commit only exports some libc functions to the Android side, which seems to be to solve the compatibility issues of some applications (bscanf/bsprintf). I agree with the changes in header files, which is pretty good to optimize header dependency, but please cautious introducing android.bp, unless you want to compile nuttx/sim into the Android environment.

Reply:
Okay, remove .bp file~